formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -311,18 +311,16 @@ and anotherClassType = {
   pub bar: int
 };
 
-class type _x =
-  [@bs]
-  {
-    pub height: int
-  };
+class type _x = {.
```

Comment:
lol! Not sure we should be giving this a sugar... but that's interesting

Reply:
@IwanKaramazow Can you explain how the `.` here impacts the perceived semantics of class types? Does it somehow "close" them in the same way that `x(.1, 2)` makes function application "closed" and how `{. x : int}` makes object types closed?

Reply:
Hmm I saw a `[@bs]` attribute & automatically implemented the syntax sugar without thinking. It felt more consistent. Anything {. } or (. ) feels more like [@bs]  now.
I can omit it.

Reply:
The OO types already kind of throw a wrench into that. I think we should make dot consistently mean “closed”, not “@bs”.

Reply:
Consider it done. 😄

Reply:
indeed, maybe we can delay this change (over class type) a bit.